code/edgebridge/build.gradle.kts

```diff
@@ -28,17 +28,18 @@ aepLibrary {
 }
 
 dependencies {
-    // TODO: Use 3.x versions for testing
-    // TODO: Remove -SNAPSHOT suffix after Core 3.0.0 is published
-    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")
```

Comment:
this should still be snapshot for now 
implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")

Reply:
The `mavenCoreVersion` in gradle.properties is set as SNAPSHOT. This will cause the POM to reference core snapshot as a dependency as well.